api/models/Burritos.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+var BurritoSchema = new Schema({
+  brandId: [{ type: Schema.Types.ObjectId, ref: 'Brand' }],
```

Comment:
This should probably not be an array as a burrito should only have one brand.

Reply:
Good point, getting rid of the arrays.